ember-cli-build.js

```diff
@@ -1,18 +1,20 @@
-/*jshint node:true*/
-/* global require, module */
-var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
-
-module.exports = function(defaults) {
-  var app = new EmberAddon(defaults, {
-    // Add options here
-  });
-
-  /*
-    This build file specifes the options for the dummy test app of this
-    addon, located in `/tests/dummy`
-    This build file does *not* influence how the addon or the app using it
-    behave. You most likely want to be modifying `./index.js` or app's build file
-  */
-
-  return app.toTree();
-};
+/*jshint node:true*/
+/* global require, module */
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+
+module.exports = function(defaults) {
+  var app = new EmberAddon(defaults, {
+    // Add options here
+  });
+
+  /*
+    This build file specifes the options for the dummy test app of this
+    addon, located in `/tests/dummy`
+    This build file does *not* influence how the addon or the app using it
+    behave. You most likely want to be modifying `./index.js` or app's build file
+  */
+
+  app.import('bower_components/blob/Blob.js');
```

Comment:
You are right! We need it in the test tree. Are you familiar with the addon hooks?

```js
// index.js
/* jshint node: true */
'use strict';

module.exports = {
  name: 'ember-cli-file-picker',
  
  included: function included(app) {
    var isProduction = app.env === 'production';

    if ( !isProduction ) {
      app.import( app.bowerDirectory + '/blob/Blob.js', { type: 'test' } );
    }

    this._super.included.apply(this, arguments);
  }
};
```

To make it work we need to add the bower dependency like here: https://github.com/funkensturm/ember-local-storage/blob/master/blueprints/ember-local-storage/index.js